src/lib/util.ts

```diff
@@ -14,20 +14,21 @@
  *  limitations under the License.
  */
 
-'use strict';
+import * as extract from 'extract-zip';
+import fetch from 'node-fetch';
+import * as fs from 'fs';
+import * as util from 'util';
+import {exec, spawn} from 'child_process';
+import * as tar from 'tar';
+import {WebManifestIcon} from './types/WebManifest';
 
-const extract = require('extract-zip');
-const fetch = require('node-fetch');
-const fs = require('fs');
-const util = require('util');
-const exec = util.promisify(require('child_process').exec);
-const tar = require('tar');
+const execPromise = util.promisify(exec);
 
-async function execute(cmd, env) {
-  await exec(cmd.join(' '), {env: env});
+export async function execute(cmd: string[], env: NodeJS.ProcessEnv): Promise<void> {
+  await execPromise(cmd.join(' '), {env: env});
```

Comment:
I feel this will fall over if any members of `cmd` have a space in them. Whether it's worth fixing or just worth documenting is up to you.

Reply:
Suppose we have something like:
`['foo', 'bar', 'foo bar]`

The output of `join` would be `foo bar foo bar`, which I think is what we expect. I'm actually using an approach with members containing spaces here: https://github.com/GoogleChromeLabs/llama-pack/blob/master/src/lib/jdk/KeyTool.js#L95-L108